### PR TITLE
Css fixes

### PR DESCRIPTION
**Pull request by asherism**

### CODE REVIEW

File: assets/svg/app/change-positive.svg [hunk 1]
```diff
@@ -1,3 +1 @@
-<svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M4 2L7 6H1L4 2Z" fill="#7FD482"/>
-</svg>
+<svg fill="none" height="4" viewBox="0 0 6 4" width="6" xmlns="http://www.w3.org/2000/svg"><path d="m2.7534.355935c.11931-.172203.37389-.172203.4932-.000001l2.02526 2.923216c.13785.19896-.00455.47085-.2466.47085h-4.050524c-.242046 0-.384443-.27189-.246599-.47085z" fill="#66dd84"/></svg>
```
Comment (Tburm) [inline assets/svg/app/change-positive.svg:1]: This color is different from the one we have set for `primaryGreen`: `#7FD482`

File: sections/dashboard/PortfolioChart/PortfolioChart.tsx [hunk 2]
```diff
@@ -37,6 +37,7 @@ const Chart = styled.div`
 	min-width: 915px;
 	border: 1px solid #353333;
 	border-radius: 16px;
+	height: 200px ;
```
Comment (Tburm) [inline sections/dashboard/PortfolioChart/PortfolioChart.tsx:40]: Is there a reason to add this height back in before we have a portfolio chart ready?

File: styles/theme/colors/common.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 const common = {
 	primaryWhite: '#ECE8E3',
-	primaryGold: '#C9975B',
+	primaryGold: '#E4B378',
```
Comment (Tburm) [inline styles/theme/colors/common.ts:5]: `#C9975B` is the color I'm seeing in Figma for `Primary Gold`. Did this change?

File: sections/dashboard/PortfolioChart/PortfolioChart.tsx (hunk 2 above)
Comment (asherism) [reply sections/dashboard/PortfolioChart/PortfolioChart.tsx:40]: just gives it some balance until then

File: styles/theme/colors/common.ts (hunk 3 above)
Comment (asherism) [reply styles/theme/colors/common.ts:5]: i tweaked it slightly for the dark theme.

File: assets/svg/app/change-positive.svg (hunk 1 above)
Comment (asherism) [reply assets/svg/app/change-positive.svg:1]: fixed

File: styles/theme/colors/common.ts (hunk 3 above)
Comment (asherism) [reply styles/theme/colors/common.ts:5]: actually gonna revert this but keep the theme specific change

**Review by Tburm — APPROVED** [review]

**Review by koredefashokun — APPROVED** [review]

**Pull request merged**